### ISSUE TITLE
Create a quest layer

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/fedbadges#17]
We want to create a spec layer on top of existing badges.  We determine a set of 5 or so badges that, once you get them all, you have completed a "quest" and receive a special quest badge for that.  When users earn such badges, people like spot and rsuehle should be emailed telling them that this has occured.  They'll then snail mail some real-life swag to the recipient.

This will also require some sort of quest-viewing interface in tahrir so people can see the quests, see their progress, etc.

**Comment by bexelbie** [comment]
resurrecting with a +1

**Comment by sayanchowdhury** [comment]
I added the functionality to tahrir, but we haven't got a way to integrate it to the UI yet. I would love to see this in the UI

**Comment by bexelbie** [comment]
Is this an admin block or a "quester" block?

If it is an admin block, can we script around it?

**Comment by justwheel** [comment]
See an example of series as they exist now:

![Series functionality in admin panel](https://user-images.githubusercontent.com/4721034/41885245-b4760614-78bc-11e8-8fc2-e52163b47ce7.png)

One thing I would like to do as part of the [Fedora Badges sysadmin workshop](https://pagure.io/flock/issue/49) is create some documentation around this, but I don't know if the UI/UX would be more important to get down first than writing documentation for these features.

**Comment by sayanchowdhury** [comment]
I think UI/UX is equally important right now, then we can go ahead and build up the series and properly show it to the contributors.

**Comment by bexelbie** [comment]
@sayanchowdhury I agree with you about the importance, but I'd also rather make the admins work a little harder once to unlock quests to be run by many.

**Comment by sayanchowdhury** [comment]
Hmm, we can this plan this up then. Everything is already there to build Quests.
